torch/_dynamo/config_utils.py

```diff
@@ -98,23 +166,48 @@ def codegen_config(self):
         """
         lines = []
         mod = self.__name__
-        for k, v in self._config.items():
-            if k in self._config.get("_save_config_ignore", ()):
+        has_saved_config_ignore = hasattr(self, "_save_config_ignore")
```

Comment:
I'm wondering if the new compile ignored subsumes save config ignore entirely

Reply:
Yeah, didn't want to disturb old code paths. 

I think some people (e.g. repro?) still use `save_config` and `load_config` which serves a different purpose...

Reply:
This can be a BE thing for later